src/main/java/org/apache/bcel/verifier/structurals/Pass3bVerifier.java

```diff
@@ -74,26 +74,68 @@
      * time, an InstructionContext object will get the current information
      * we have about its symbolic execution predecessors.
      */
-    private static final class InstructionContextQueue{
+    private static final class InstructionContextQueue {
+        // The following two fields together represent the queue.
+        /** The first elements from pairs in the queue. */
         private final List<InstructionContext> ics = new Vector<>();
+        /** The second elements from pairs in the queue. */
         private final List<ArrayList<InstructionContext>> ecs = new Vector<>();
+
+        /**
+         * Add an (InstructionContext, ExecutionChain) pair to their respective queues.
```

Comment:
Hi @mernst 
Thank you chipping in. I am trying to get Commons to be consistent in Javadocs to be in the form "Adds ..."., "Removes ...", "Gets ...", "Sets ...", and so on.
TY!